imap_processing/cli.py

```diff
@@ -77,8 +79,8 @@ def _parse_args():
 
     Returns
     -------
-    args : argparse.Namespace
-        An object containing the parsed arguments and their values
+    args :
+        (argparse.Namespace) An object containing the parsed arguments and their values.
```

Comment:
I'm surprised this one was failing. It looks like that should have been right with including both the name and return type?
https://numpydoc.readthedocs.io/en/latest/format.html#returns

```suggestion
    argparse.Namespace
        An object containing the parsed arguments and their values.
```

Reply:
I have no idea why this one fails. Ive tried messing around with it and have had no luck. My only guess is that the readthedocs are for older versions. (This also occurs with the first `"""`. The checks want it on its own line when it is a multi line docstring). It was quite frustrating in the beginning because it did not follow the Parameters section.